conveyor/src/main/java/com/enfint/conveyor/dto/EmploymentDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.enfint.conveyor.dto;
+
+import com.enfint.conveyor.enumModel.EmploymentStatus;
+import com.enfint.conveyor.enumModel.Position;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class EmploymentDTO {
+    private Enum<EmploymentStatus> employmentStatus;
+    private String employerINN;
+    private BigDecimal salary;
+    private Enum<Position> position;
```

Comment:
Here and in another places: you can use Position enum directly.  `private Position position;` instead of `Enum<Position> position;`